tests/acceptance/suites/vertical-full-page-map-with-filters.js

```diff
@@ -3,12 +3,18 @@ import SearchBar from '../blocks/searchbar';
 import VerticalResults from '../blocks/verticalresults';
 import ThemeMap from '../blocks/thememap';
 import CollapsibleFilters from '../blocks/collapsiblefilters';
+import { registerIE11NoCacheHook } from '../../test-utils/testcafe';
 
 fixture`Vertical Full Page Map with Filters and Clusters`
   .page(`http://localhost:${PORT}/locations_full_page_map_with_filters`)
+  .beforeEach(async t => {
+    await VerticalResults.registerLogger(t);
+    await registerIE11NoCacheHook(t);
```

Comment:
should the ie11 hook go inside registerLogger? like, is there ever a time we would call registerLogger without also registering the ie11 hook?

Reply:
yeah, that would make sense. Updated!